<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaGalerias</class>
 <widget class="QWidget" name="ventanaGalerias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="fondoG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>563</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreGaleria">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>150</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="nombreLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombre de la galer√≠a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volverButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>volver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agregarButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>agregar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
